src/main/java/uk/gov/pay/webhooks/deliveryqueue/managed/SendAttempter.java

```diff
@@ -70,8 +72,15 @@ public void attemptSend(WebhookDeliveryQueueEntity queueItem) {
         var retryCount = webhookDeliveryQueueDao.countFailed(queueItem.getWebhookMessageEntity());
         Instant start = instantSource.instant();
 
+        URL url = null;
+        try {
+            url = new URL(webhook.getCallbackUrl());
+        } catch (MalformedURLException e) {
+            LOGGER.info(format("Callback URL %s is malformed.", webhook.getCallbackUrl()));
```

Comment:
Should this be a `WARN`|`ERROR` rather than `INFO`?

Reply:
I suppose ERROR is appropriate. There is validation on the selfservice UI though so this really shouldn't happen.